provenance/assetlist.json

```diff
@@ -19,15 +19,15 @@
       "display": "hash",
       "symbol": "HASH",
       "logo_URIs": {
-        "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/provenance/images/prov.png",
-        "svg": "https://raw.githubusercontent.com/cosmos/chain-registry/master/provenance/images/prov.svg"
+        "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/hastra/images/hastra.png",
+        "svg": "https://raw.githubusercontent.com/cosmos/chain-registry/master/hastra/images/hastra.svg"
       },
       "images": [
         {
-          "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/provenance/images/prov.png",
-          "svg": "https://raw.githubusercontent.com/cosmos/chain-registry/master/provenance/images/prov.svg",
+          "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/hastra/images/hastra.png",
+          "svg": "https://raw.githubusercontent.com/cosmos/chain-registry/master/hastra/images/hastra.svg",
```

Comment:
Sorry to nit pick, but these images have different amounts of padding, which will cause them to look different based on whether the svg vs png is used. Either these should be separated, or better yet ensure to use images with identical padding.

Reply:
Not an issue. I'll get updated images and get those submitted. Appreciate the review.